.github/nightly_matrix.php

```diff
@@ -46,26 +46,17 @@ function get_matrix_include(array $branches) {
     $jobs = [];
     foreach ($branches as $branch) {
         $jobs[] = [
-            'name' => '_ASAN_UBSAN',
+            'name' => '_ASAN_UBSAN_REPEAT',
             'branch' => $branch,
             'debug' => true,
             'zts' => true,
             'configuration_parameters' => "CFLAGS='-fsanitize=undefined,address -DZEND_TRACK_ARENA_ALLOC' LDFLAGS='-fsanitize=undefined,address'",
             'run_tests_parameters' => '--asan',
-            'test_function_jit' => false,
```

Comment:
Note that function JIT was specifically disabled on ASAN because it exceeded the 6h limit on GitHub Actions.